rotkehlchen/assets/exchanges_mappings/kucoin.py

```diff
@@ -228,4 +228,11 @@
     evm_address_to_identifier('0xEec2bE5c91ae7f8a338e1e5f3b5DE49d07AfdC81', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'DPX',  # noqa: E501
     evm_address_to_identifier('0xE55d97A97ae6A17706ee281486E98A84095d8AAf', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'AIPAD',  # noqa: E501
     evm_address_to_identifier('0x12970E6868f88f6557B76120662c1B3E50A646bf', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'LADYS',  # noqa: E501
+    evm_address_to_identifier('0x628A3b2E302C7e896AcC432D2d0dD22B6cb9bc88', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'LMWR',  # noqa: E501
+    evm_address_to_identifier('0x72aDadb447784dd7AB1F472467750fC485e4cb2d', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'WLD',  # noqa: E501
```

Comment:
again same here. I think youare mapping worldcoing wrong. This is WLD: https://etherscan.io/address/0x163f8c2467924be0ae7b5347228cabf260318753